src/main/java/com/minecolonies/coremod/commands/colonycommands/requestsystem/RSResetCommand.java

```diff
@@ -0,0 +1,109 @@
+package com.minecolonies.coremod.commands.colonycommands.requestsystem;
+
+import com.minecolonies.api.colony.IColony;
+import com.minecolonies.api.colony.permissions.Rank;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.colony.permissions.Rank'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)